src/translations/en/publicationTypes.json

```
@@ -2,19 +2,28 @@
   "Book": "Book",
   "BookAnthology": "Anthology",
   "BookMonograph": "Monograph",
-  "Chapter": "Chapter",
-  "Degree": "Degree",
-  "DegreeBachelor": "Bachelor",
-  "DegreeMaster": "Master",
-  "DegreePhd": "PhD",
-  "Journal": "Publication in Journal",
+  "BookChapter": "Chapter in anthology",
```

her går det i ball med hvordan det er på backend atm da, @sarahse ... 
dette tar utgangspunkt i hvordan det skal bli, usikker på hvordan vi skal håndtere det (eller om vi i det hele tatt skal)

---

Kan si med en gang at det der ikke blir stående 😅  jeg legger jo til kapittel som type nå, så det må nok `ChapterArticle`
```suggestion
  "ChapterArticle": "Chapter in Anthology",
```